### ISSUE TITLE
process (unmarshal) raw objects ?

### ISSUE THREAD

**Issue by fenugrec** [zrax/pycdc#428]
Possibly newb question, new to py reversing. I have this blob (most likely `__init__.py(c)` that is then passed to PyMarshal_ReadObjectFromString() ; not sure what happens then and I'm hoping pycdc can work on that.

First few bytes of init.bin :

```
00 00 00 00 E3 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 00 09 00 00 00 40 00 00 00 73 80 04 00 00 7A 16 64 00 64 01 84 00 6A 00 01 00 65 01 64 02 83 01 82 01 57 00 6E 12
```

trying to run pycdc on that gives "bad magic", and the caveman method of  trying to tweak/ insert/delete some bytes at the beginning of that file, does not help.
Any tips are most welcome.
Thanks !

**Comment by greenozon** [comment]
![image](https://github.com/zrax/pycdc/assets/765343/2450ca78-a3f2-4139-a5ce-10b735a9c37c)


bad magic means your bytecode pyc might be encrypted/obfuscated/whatever not suitable for vanilla python guy

**Comment by fenugrec** [comment]
> bad magic means your bytecode pyc might be encrypted/obfuscated/whatever not suitable for vanilla python guy


Thanks. In fact I had already decrypted it; there is some plaintext inside. I eventually managed to graft enough header bytes from a plaintext .pyc file, that now pycdc at least attemps a decompile, but aborts with ` Unsupported opcode: JUMP_IF_NOT_EXC_MATCH`  , i.e. https://github.com/zrax/pycdc/issues/242 .

Seems like py3.9 + is really not well supported by any python decompiler yet...
